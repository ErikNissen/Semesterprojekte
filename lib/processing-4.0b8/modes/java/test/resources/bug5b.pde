version https://git-lfs.github.com/spec/v1
oid sha256:e8eacf628d9364ac3a59431fb769f2ae0bc8858dd83a4adfa3a75d980b3beaa9
size 79
