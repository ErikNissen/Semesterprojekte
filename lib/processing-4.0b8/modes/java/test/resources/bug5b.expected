version https://git-lfs.github.com/spec/v1
oid sha256:04890c9bb0c95b6c54a0476fa44a39caa9cd44e05cc6f894632d185867d563ea
size 769
